modules/administration_manual/pages/release_notes.adoc

```diff
@@ -17,6 +18,65 @@
+
+=== WebDAV Locks
+
+ownCloud Server 10.1 introduces WebDAV Locks that allow clients to lock and unlock resources to prevent other users from making changes. The feature has been implemented as a prerequisite for manual file locking and MS Office Online Server compatibility. In the current state, file locking is only available via API. As in some cases file locks may unintentionally stay active if there is a default lock timeout of **XX**. Users can recognize locked files via the "lock" icon in the file list. Additionally a lock owner (the user who locked the file) can manually unlock them via the "Locks" tab in the right sidebar. The "Locks" tab will only appear for files that have active locks.
+
+=== Federation: Compliance with proposed OpenCloudMesh 1.0 specification
+
+Federation enables instances of ownCloud and other supporting platforms to exchange information. It allows users to share data across installations building a worldwide collaboration network of decentralized nodes - each under the full control of it's provider. Together with the other vendors the underlying OpenCloudMesh API specification has been shifted to a new level to clean up the interface, improve it's stability and to set the foundation for future feature improvements. ownCloud Server 10.1 is compliant with the new specification proposal. The introduction of the new specification does not involve changes in functionality for users.
+
+=== New Collaborative Tags Scope: Static Tags
+
+Version 10.1 comes with a new scope for Collaborative Tags called "Static Tags". In addition to the https://doc.owncloud.com/server/10.0/user_manual/files/webgui/tagging.html#filter-by-tag[other tag scopes], these tags are intended to be supplied by administrators and linked with policies in the https://marketplace.owncloud.com/apps/firewall[File Firewall], https://marketplace.owncloud.com/apps/files_classifier[Document Classification] or https://marketplace.owncloud.com/apps/workflow[Workflows], for example. Every user will be able to see these tags assigned to files but only users in specified groups have the permission to assign or unassign them. This makes it possible to equip certain users with the means to impose pre-defined policies upon files. To create such tags administrators need to use the https://marketplace.owncloud.com/apps/systemtags_management[Collaborative Tags Management] extension.
+
+=== Other notable changes
+
+- The user/group deletion in the users page now has a confirmation dialog to prevent unintentional user deletion
+- The default public link share name has been changed to be "Public link" instead of formerly the file or folder's name
+- **Allow loading JSON files in setups with pretty URLs**
```

Comment:
"to make sure this", sentence not ended ?

Reply:
Yeah, because I need input from you on that one ;)

Reply:
Something like "Please check that the htaccess file has updated automatically. If not, see https://github.com/owncloud/core/pull/32718/files for the required change"

Reply:
Thanks, added.